lib/api/web-element/commands/click.js

```diff
@@ -19,9 +19,9 @@
  * @memberof ScopedWebElement
  * @instance
  * @syntax browser.element(selector).click()
- * @see https://www.w3.org/TR/webdriver#dfn-element-click
+ * @link /#dfn-element-click
  * @returns {ScopedWebElement}
  */
-module.exports.command = function() {
+module.exports.command = function () {
```

Comment:
I suggest, remove this unnecessary space between as it shows no significant change.